components/collapsible-panel/collapsible-panel.js

```diff
@@ -310,6 +311,12 @@ class CollapsiblePanel extends SkeletonMixin(FocusMixin(RtlMixin(LitElement))) {
 			.d2l-collapsible-panel:not(.has-summary) .d2l-collapsible-panel-summary {
 				display: none;
 			}
+
+			@media (prefers-contrast: more) {
+				d2l-icon-custom svg {
+					color: inherit;
```

Comment:
I was able to get this working without this block and the change on line 254.. so just the adjustment to the SVG and line 259. But I could definitely be missing something!